README.md

```diff
@@ -21,47 +21,45 @@ crates:
 
 ## Building
 
-Ensure you have a recent Rust toolchain installed. You can compile the entire
-workspace from the repository root:
+Ensure you have a recent Rust toolchain installed, then build the entire workspace:
 
 ```bash
 cargo build
 ```
 
-Each crate can also be built individually. The UI depends on the core library,
-so building the frontend will build the core automatically:
+For better performance when playing games, use a release build:
 
 ```bash
-# Core library only
-cargo build -p vibe-emu-core
-
-# UI frontend (pulls in the core crate as a dependency)
-cargo build -p vibe-emu-ui
+cargo build --release
 ```
 
-The frontend uses `winit` with the `pixels` crate for window creation and
-rendering via `wgpu`. On Linux you may need X11 development packages installed
-(e.g. `libx11-dev`) and GTK development headers (`libgtk-3-dev`). Audio output
-relies on `cpal`, which requires ALSA headers. Install `libasound2-dev` as well
-if you build on Linux.
+### Platform Requirements
+
+- **Windows**: Visual Studio Build Tools with C++ support
+- **Linux**: X11, GTK3, and ALSA development packages
+- **macOS**: Xcode Command Line Tools
+
+For detailed platform-specific instructions, troubleshooting, and build configuration options, see [BUILD.md](BUILD.md).
 
 ### Mobile Adapter GB support (bundled by default)
 
 The UI builds with **Mobile Adapter GB** support enabled by default using the
 vendored `vendor/libmobile-0.2.2` sources. This requires a working C toolchain
 on your platform (e.g. MSVC Build Tools on Windows, or clang/gcc on Linux/macOS).
 
-If you want to build the UI without Mobile support:
+**License Notice**: The bundled `libmobile` library is licensed under the **GNU Lesser General Public License (LGPL) v3**. See `vendor/libmobile-0.2.2/COPYING.LESSER` for the full license text. The LGPL permits linking this library into your application. If you wish to use a modified or updated version of `libmobile`, you have two options:
 
-```bash
-cargo build -p vibe-emu-ui --no-default-features
-```
+1. **Use the system-provided library**: Build vibeEmu with the `mobile-system` feature to link against your own `libmobile` installation instead of the bundled version:
+   ```bash
+   cargo build -p vibe-emu-ui --no-default-features --features mobile-system
+   ```
```

Comment:
but "how" does one use a system provided library. What does this *mean*? Do they need it located somewhere specific? Named something specific? compiled already? ect.
Needs to be clear for a non-technical user.